starknet_devnet/__init__.py

```diff
@@ -42,4 +44,9 @@ def simpler_copy(self, memo):  # pylint: disable=unused-argument
     return copy(self)
 
 
+# Apply cairo-rs-py patch
```

Comment:
Lines 47-51 are inserted in the middle of another monkey patch (the definition of `simpler_copy` and its setting)

Reply:
My bad! I moved it now